test/trip_plan/api/open_trip_planner/parser_test.exs

```diff
@@ -19,8 +19,20 @@ defmodule TripPlan.Api.OpenTripPlanner.ParserTest do
       end
 
       assert [first, _, _] = parsed
-      assert first.start == Timex.to_datetime(~N[2017-05-19T13:50:59], "America/New_York")
-      assert first.stop == Timex.to_datetime(~N[2017-05-19T14:03:19], "America/New_York")
+
+      is_equal_time? = fn t1, t2 ->
+        DateTime.compare(t1, t2) == :eq
```

Comment:
updated because this comparator was more forgiving - without it I was getting silly failures such as this:

```
Assertion with == failed
     code:  assert first.start == Timex.to_datetime(~N"2017-05-19T13:50:59", "America/New_York")
     left:  #DateTime<2017-05-19 13:50:59.000-04:00 EDT America/New_York>
     right: #DateTime<2017-05-19 13:50:59-04:00 EDT America/New_York>
```